### PR TITLE
Updates nav-bar with new links and drop-downs

### PR DESCRIPTION
**Pull request by thesuss**
PT Story: 
https://www.pivotaltracker.com/story/show/131922325

Changes proposed in this pull request:
- More links in menu for restaurant owners

What I have learned working on this feature: 
- KISS =)

Screenshots: 
skarmavbild 2016-10-07 kl 23 49 30

Ready for review!

### CODE REVIEW

File: app/views/application/_navbar.html.haml [hunk 1]
```diff
@@ -11,18 +11,38 @@
...
-            %li
-              = link_to 'Create menu', new_menu_path
-            %li
-              = link_to 'Create dish', new_dish_path
-        %li
-          - unless current_user.try(:owner?)
+        - if current_user
+          %li
+            = link_to 'Your profile', edit_user_registration_path
+        - if current_user.try(:owner?)
```
Comment (tochman) [inline app/views/application/_navbar.html.haml]: Again, we are using abilities for this kind of conditions. Please go over the documentation for CanCanCan.

File: app/views/application/_navbar.html.haml [hunk 2]
```diff
@@ -11,18 +11,38 @@
...
+              Restaurant
+              %strong.caret
+            %ul.dropdown-menu
+              %li
+                = link_to 'Show your restaurant', restaurant_path(current_user)
+              %li
+                = link_to 'Create a new restaurant', new_restaurant_path(current_user)
+              %li
+                = link_to 'Update your restaurant', edit_restaurant_path(current_user)
+          - if current_user.restaurant
```
Comment (tochman) [inline app/views/application/_navbar.html.haml]: Same as above

File: app/views/application/_navbar.html.haml [hunk 3]
```diff
@@ -11,18 +11,38 @@
...
+                %li
+                  = link_to 'Create a menu', new_menu_path(current_user.restaurant)
+            %li.dropdown
+              = link_to '#', {class: 'dropdown-toggle', data: {toggle: 'dropdown'}} do
+                Dishes
+                %strong.caret
+              %ul.dropdown-menu
+                %li
+                  = link_to 'Create a dish', new_dish_path(current_user.restaurant)
+        - unless current_user.try(:owner?)
```
Comment (tochman) [inline app/views/application/_navbar.html.haml]: And again

**Review by tochman — CHANGES_REQUESTED** [review]

**Comment by thesuss** [comment]
I have refactored it, and this time it did not affect the test nor the user experience

**Review by tochman — APPROVED** [review]

**Pull request merged**